src/CronJobListBuilder.php

```diff
@@ -57,10 +58,32 @@ public function buildRow(EntityInterface $entity) {
     // In milliseconds.
     $row['duration'] = round(($entry->end_time - $entry->start_time) * 1000, 0);
     $row['status'] = $entity->status() ? $this->t('Yes') : $this->t('No');
+
     return $row + parent::buildRow($entity);
   }
 
   /**
+   * Returns the default operations for the entity.
+   *
+   * @param EntityInterface $entity
+   *   The entity of this row.
+   *
+   * @return array
+   *   The array structure is identical to the return value of
+   *   parent::getOperations().
+   */
+  public function getDefaultOperations(EntityInterface $entity) {
+    $link = Url::fromUri('internal:/admin/config/system/cron/jobs/' . $entity->id() . '/run');
```

Comment:
Try using the route for this. Or, register it as a link template, then you can just do $entity->urlInfo('run') or so.